source/Calamari.Shared/Commands/ApplyDeltaCommand.cs

```diff
@@ -24,13 +26,15 @@ public class ApplyDeltaCommand : Command
         bool skipVerification;
         
         readonly IFreeSpaceChecker freeSpaceChecker;
+        readonly IVariables variables;
         readonly ICalamariFileSystem fileSystem;
         readonly ICommandLineRunner commandLineRunner;
         readonly ILog log;
 
-        public ApplyDeltaCommand(ILog log, IFreeSpaceChecker freeSpaceChecker, ICalamariFileSystem fileSystem, ICommandLineRunner commandLineRunner)
+        public ApplyDeltaCommand(ILog log, IFreeSpaceChecker freeSpaceChecker, IVariables variables, ICalamariFileSystem fileSystem, ICommandLineRunner commandLineRunner)
```

Comment:
IIRC variables was added to get the task id? Is it still needed?

Reply:
Not needed, good pickup 👍